src/main/java/org/terasology/blockNetwork/BlockNetworkUtil.java

```diff
@@ -40,6 +53,13 @@ private static Rotation getRotation(Block block) {
         return rotation;
     }
 
+    /**
+    * Give the new side of the block based on its' rotation
+    *
+    * @param block - the block that is about to be checked its' rotation and counted its' connections
```

Comment:
"The block _whose_ rotation is to be checked and whose connections are to be counted"